fdbclient/FDBTypes.h

```diff
@@ -1126,4 +1126,27 @@ inline const char* transactionPriorityToString(TransactionPriority priority, boo
 	throw internal_error();
 }
 
+struct VersionVector {
+	std::map<Tag, Version> versions;
+
+	VersionVector() {}
+
+	void setVersion(const Tag& tag, Version version) {
+		versions[tag] = version;
+	}
+
+	bool hasVersion(const Tag& tag) const {
+		return versions.find(tag) != versions.end();
```

Comment:
`std::contains` ?

Reply:
I see std::includes but not std::contains? Is that a generic std method?

Reply:
I think `std::map::contains` is a thing, but it was introduced in C++20 (which we don't use yet). Alternatively you could use `std::map::count`, though personally I prefer your usage of `find` as it seems cleaner.

Reply:
Oh, thanks for that info!